QuestionBot/QuestionBot.UnitTests/Model/StoreTests.cs

```diff
@@ -1,13 +1,77 @@
-﻿using NUnit.Framework;
```

Comment:
minor: This file should be called InMemoryStoreTests.cs to match the other file.